velox/docs/configs.rst

```diff
@@ -335,6 +331,12 @@ Spilling
      - boolean
      - true
      - When `spill_enabled` is true, determines whether Window operator can spill to disk under memory pressure.
+   * - window_spill_min_read_batch_rows
+     - integer
+     - 1000
+     - When processing spilled window data, read batches of whole partitions having at least that many rows. Set to 1 to
```

Comment:
@HolyLow : I had been following this review, but saw the final commit today. 

This is a bit misleading... Are you saying that we read either one full partition > 1000 rows or as many partitions that fit in 1000 rows. Though it seems like we could have cases where we have x partitions taking 999 rows but then the last partition read for the last row could have > 1000 rows so we eventually read 2000 rows  maybe ?

Also why are you saying set to 1 ? Didn't follow that part.

@mbasmanova

Reply:
@aditi-pandit Aditi, you are correct. We read one whole partition + n more partitions until we read at least 1000 rows. We may read more rows than 1000. 

> Set to 1 ...

This is advice to the user. They can set this config to 1 to ensure no more than 1 whole partition is read at a time. This would be the original behavior.